### PR TITLE
 [MIG][12.0] helpdesk_ticket_type_default_priority: Migrated.

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Helpdesk Ticket Type Default Priority
============================
Set the priority based on the type.

### CODE REVIEW

**Review by osi-scampbell — COMMENTED** [review]
@murtuzasaleh Code looks great, installs cleanly and the functionality works. The travis build is failing because we haven't updated the travis.yml file for v12 can you update it please?

**Comment by max3903** [comment]
@murtuzasaleh Travis cannot test modules depending on Enterprise modules (like helpdesk), so those modules needs to be excluded from the tests. 

See how to do that in v11 here:
https://github.com/ursais/osi-addons/blob/11.0/.travis.yml#L28

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh Travis cannot test modules depending on Enterprise modules (like helpdesk), so those modules needs to be excluded from the tests.
> 
> See how to do that in v11 here:
> https://github.com/ursais/osi-addons/blob/11.0/.travis.yml#L28

@max3903 So can I copy and paste this line in v12 and Create PR of it?

**Comment by max3903** [comment]
@murtuzasaleh No need for another pull request. As part of the module migration, it should include the update to the exclude list in .travis.yml

For example, in this pull request, you should add `helpdesk_ticket_type_default_priority` in the exclude list.

**Comment by max3903** [comment]
Same idea with `oca_dependencies.txt`. It should be updated with the pr of the migrated module that needs it.

**Comment by murtuzasaleh** [comment]
> Same idea with `oca_dependencies.txt`. It should be updated with the pr of the migrated module that needs it.

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh No need for another pull request. As part of the module migration, it should include the update to the exclude list in .travis.yml
> 
> For example, in this pull request, you should add `helpdesk_ticket_type_default_priority` in the exclude list.

Sure, Thanks

**Comment by murtuzasaleh** [comment]
@max3903 @osi-scampbell I have improved travis file and added oca_dependencies.txt file, is it right?

File: .travis.yml [hunk 1]
```diff
@@ -21,8 +21,8 @@ env:
 
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-  - TESTS="1" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - TESTS="1" ODOO_REPO="OCA/OCB" LINT_CHECK="0" EXCLUDE="mrp_barcode,stock_picking_no_over_scan,helpdesk_ticket_to_task,helpdesk_fieldservice,agreement_helpdesk,helpdesk_stock,helpdesk_scope,helpdesk_ticket_type_default_priority,helpdesk_phone,fieldservice_helpdesk_stock,agreement_sale_subscription,helpdesk_ticket_parent,helpdesk_fieldservice_agreement,helpdesk_resolution"
+  - TESTS="1" ODOO_REPO="odoo/odoo" LINT_CHECK="0" EXCLUDE="mrp_barcode,stock_picking_no_over_scan,helpdesk_ticket_to_task,helpdesk_fieldservice,agreement_helpdesk,helpdesk_stock,helpdesk_scope,helpdesk_ticket_type_default_priority,helpdesk_phone,fieldservice_helpdesk_stock,agreement_sale_subscription,helpdesk_ticket_parent,helpdesk_fieldservice_agreement,helpdesk_resolution"
```
Comment (max3903) [inline .travis.yml]: Just add the `helpdesk_ticket_type_default_priority` module to the list.

File: oca_dependencies.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+contract
+field-service
+geospatial
+odoo-addons https://github.com/ursais/odoo-addons.git 12.0
+stock-rma https://github.com/Eficent/stock-rma.git 12.0
+
```
Comment (max3903) [inline oca_dependencies.txt]: We don't need `oca_dependencies.txt` for `helpdesk_ticket_type_default_priority`. Please remove the file.

**Review by max3903 — APPROVED** [review]

**Pull request merged**